test/smoke/testApps/SimpleEventData/src/main/java/com/microsoft/ajl/simplecalc/CalculationHistoryService.java

```diff
@@ -0,0 +1,141 @@
+package com.microsoft.ajl.simplecalc;
```

Comment:
This file was in progress when I left this test application. It should be deleted.